linkerd2-cni.tf

```diff
@@ -16,22 +16,6 @@ locals {
   )
 
   values_linkerd2-cni = <<VALUES
-    namespace: ${local.linkerd2-cni.namespace}
```

Comment:
@cebidhem this is not required anymore ?

Reply:
It's not indeed. But it has been removed already in the branch, looking at why it doesn't appear in the PR...